tests/foreman/cli/test_host.py

```diff
@@ -66,6 +67,38 @@
 from robottelo.hosts import ContentHostError
 
 
+@pytest.fixture(scope="function")
+def rhel_version():
+    current_test_name = os.environ.get('PYTEST_CURRENT_TEST')
+    start = current_test_name.find('[') + 1
+    end = current_test_name.find(']')
```

Comment:
start and end used just because of black formatting and flake.
black wants:
`[current_test_name.find('[') + 1 : current_test_name.find(']')]`
flake didn't like the space after `1`